build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     testCompile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testCompile group: 'net.sf.kxml', name: 'kxml2', version: '2.3.0'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
```

Comment:
Junit4 includes `hamcrest-core` which is a very basic library with `assertThat` and some basic matchers only. I'm adding `hamcrest-all`, which includes all sorts of useful matchers that we would have to implement ourselves otherwise.